integration/cmd_type_test.go

```diff
@@ -152,6 +152,21 @@ func TestGenerateAnAppWithStargateWithTypeAndVerify(t *testing.T) {
 		ExecShouldError(),
 	))
 
+	env.Must(env.Exec("create a type with no interaction message",
+		step.NewSteps(step.New(
+			step.Exec("starport", "type", "nomessage", "email", "--no-message"),
+			step.Workdir(path),
+		)),
+	))
+
+	env.Must(env.Exec("should prevent creating a legacy type with no message",
+		step.NewSteps(step.New(
+			step.Exec("starport", "type", "nomessagelegacy", "--no-message", "--legacy"),
```

Comment:
Do we need to support -legacy at all?

Reply:
Because we still have `--legacy` option, I can remove this test now that we know we'll stop supporting this